src/app/shell/result/workshop-map-view-list/workshop-map-view-list.component.ts

```diff
@@ -49,17 +51,24 @@ export class WorkshopMapViewListComponent implements OnInit, OnDestroy {
   };
   public workshopDetailsAnimationState = false;
 
-
   @ViewChild('WorkshopsWrap') workshopsWrap: ElementRef;
   @ViewChild('CurSelectedWorkshop') curSelectedWorkshop: ElementRef;
   widthOfWorkshopCard = Constants.WIDTH_OF_WORKSHOP_CARD;
 
-
   private swipeCoord?: [number, number];
   private swipeTime?: number;
   public currentWorkShopIndex: number = 0;
   public direct: string
   public left: number = 0
+
+  constructor(private store: Store) { }
+
+  ngOnInit(): void {
+    this.filteredWorkshops$
+      .pipe(takeUntil(this.destroy$), filter((filteredWorkshops) => !!filteredWorkshops))
```

Comment:
.pipe(
  takeUntil(this.destroy$), 
  filter((filteredWorkshops) => !!filteredWorkshops)
)